lib/convection/control/stack.rb

```diff
@@ -248,6 +275,22 @@ def validate
         puts "\nTemplate validated successfully"
       end
 
+      def after_create_task(task)
+        @tasks[:after_create].unshift(task)
```

Comment:
For after tasks, I disagree. Before tasks are run in the order you register them. I've modelled tasks in a manner similar to how test hooks are dealt with in several unit testing frameworks. If you have cleanup you do it in reverse order, from setup, in my experience.